### PR TITLE
Add dedicated batch session idle timeout and prevent the batch submission operation timeout

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Prevent batch session & operation closed by kyuubi because of timeout
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by ulysses-you** [comment]
would batch session leak if we disable timeout ?

**Comment by turboFei** [comment]
> would batch session leak if we disable timeout ?
It will timeout after the operation terminated.

File: kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala [hunk 1]
```diff
@@ -288,6 +288,19 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
...
+          warn(s"Error closing idle session ${session.handle}", e)
+      }
+    } else {
+      session.closeExpiredOperations()
+    }
+  }
+
   private def startTimeoutChecker(): Unit = {
     val interval = conf.get(SESSION_CHECK_INTERVAL)
     val timeout = conf.get(SESSION_IDLE_TIMEOUT)
```
Comment (ulysses-you) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala:306]: How about add a new filed `sessionTimeout` in `Session`. Use  `SESSION_IDLE_TIMEOUT ` by default in `AbstractSession`, and use `BATCH_SESSION_IDLE_TIMEOUT` in `KyuubiBatchSessionImpl`.

Then we can make each session use it's own timeout threshold.

Comment (turboFei) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala:306]: thanks, will address it

**Review by ulysses-you — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to master and 1.6.2

**Pull request closed without merging**